tests/e2e-openshift/monitoring/check_metrics.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Could this shellscript be started in a container (like in the verify-traces step)? This will give us more deterministic results (e.g. curl might not be installed on every test host, may have a different version, etc.) and remove dependencies of the test system.